.gitignore

```diff
@@ -0,0 +1,2 @@
+# jetbrains
+.idea/
```

Comment:
Could you add this to your global gitignore? We don't want proliferation of gitignore files everywhere for different IDEs and all.

Reply:
Ok. Should I remove it from [go-libp2p](https://github.com/libp2p/go-libp2p/commit/ce95b37cd31edb3d9e4a4458cb43be0c9c77704f#diff-a084b794bc0759e7a6b77810e01874f2) as well?

Reply:
no need